connect_ext_ppr/utils.py

```diff
@@ -87,6 +89,10 @@ def get_hubs(client, hubs_ids):
     return client.hubs.filter(rql)
 
 
+def get_partners(client, account_ids):
```

Comment:
[COD-C] It will not work for the syndication products, since there is no way to know the vendor. Why you don't want to extract that information from the product itself? I think it contains owner id and name

Reply:
Or even store in DB the owner also? Just two plain fields